File: ARCADE/Intro/Level 4_1: alternatingSums.cs
    using System;
     
    public class Test
    {
    	public static int[] alternatingSums(int[] a){
            int sum1 = 0, sum2 = 0;
            for (int i = 0; i < a.Length; i++){
                if (i % 2 == 0) sum1 += a[i];
                else sum2 += a[i];
            }
            int[] result = { sum1, sum2 };
            return result;
        }
    	public static void Main()
    	{
    		int [] a = new int []{50, 60, 60, 45, 70};
    		var b = alternatingSums(a);
    		foreach(int i in b){
    			Console.Write(i + " ");
    		}
    	}
    }

File: ARCADE/Intro/Level 4_1: alternatingSums.pas
program alternatingSums;

Type
	ArrayInteger = Array of integer;

function alternatingSums(a: ArrayInteger): ArrayInteger;
var arr : Array of integer;
    i, sum1, sum2 : integer;
begin
    sum1 := 0; sum2 := 0;
    for i := 0 to length(a) - 1 do
        if(i mod 2 = 0) then sum1 := sum1 + a[i]
        else sum2 := sum2 + a[i];
    SetLength(arr, 2);
    arr[0] := sum1;
    arr[1] := sum2;
    alternatingSums := arr;
end;

var a, b : ArrayInteger;
	i : integer;
	
begin
	a := ArrayInteger.Create(50, 60, 60, 45, 70);
	b := alternatingSums(a);
	for i := 0 to length(b) - 1 do 
		writeln(b[i]);
end.
